spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -403,43 +404,25 @@ def fixture_record(fixture_name)
       expect(@hathi_permanent['hathi_identifier_s']).to contain_exactly("mdp.39015036879529")
     end
   end
-  # TODO: Replace with Alma
-  describe 'holdings_1display' do
-    # before(:all) do
-    #   marcxml = fixture_record('7617477')
-    #   @solr_hash = @indexer.map_record(marcxml)
-    #   @holding_block = JSON.parse(@solr_hash['holdings_1display'].first)
-    #   holdings_file = File.expand_path("../../../fixtures/marc_to_solr/7617477-holdings.json", __FILE__)
-    #   holdings = JSON.parse(File.read(holdings_file))
-    #   @holding_records = []
-    #   holdings.each { |h| @holding_records << MARC::Record.new_from_hash(h) }
-    # end
 
-    # TODO: Replace with Alma
-    # Revisit while working on https://github.com/pulibrary/marc_liberation/issues/921
-    xit 'groups holding info into a hash keyed on the mfhd id' do
-      @holding_records.each do |holding|
-        holding_id = holding['001'].value
-        expect(@holding_block[holding_id]['location_code']).to include(holding['852']['b'])
-        expect(@holding_block[holding_id]['location_note']).to include(holding['852']['z'])
-      end
-    end
-    # TODO: Replace with Alma
-    # Revisit while working on https://github.com/pulibrary/marc_liberation/issues/921
-    xit 'includes holding 856s keyed on mfhd id' do
-      @holding_records.each do |holding|
-        holding_id = holding['001'].value
-        electronic_access = @holding_block[holding_id]['electronic_access']
-        expect(electronic_access[holding['856']['u']]).to include(holding['856']['z'])
-      end
+  describe 'holdings_1display' do
+    it 'groups holding info into a hash keyed on the mfhd id' do
+      @holdings = JSON.parse(@sample37["holdings_1display"][0])
+      expect(@holdings.keys).to eq ["22170509880006421", "22170509890006421", "22170509900006421"]
+      expect(@holdings['22170509880006421']['location_code']).to eq 'engineer$stacks'
+      expect(@holdings['22170509890006421']['location_code']).to eq 'lewis$stacks'
+      expect(@holdings['22170509900006421']['location_code']).to eq 'firestone$stacks'
+      # expect(@holding_block[holding_id]['location_note']).to include(holding['852']['z'])
     end
```

Comment:
I would like to keep this comment here until I get an answer on my question regarding the location_note from @mzelesky .